src/payload.config.ts

```diff
@@ -19,6 +19,7 @@ export default buildConfig({
   collections: [Users, Events, Media],
   typescript: {
     outputFile: path.resolve(__dirname, 'payload-types.ts'),
+    declare: false,
```

Comment:
What does this do for us? Could you please add a small comment with an explanation?

Reply:
https://payloadcms.com/docs/typescript/generating-types#disable-declare-statement
Before that I always manually removed the declare statement after copying it over to the website repo.